Slash/Utility/Comments/Comments.pm

```diff
@@ -1157,37 +785,31 @@ sub printComments {
 	# Flat and theaded mode don't index, even on large stories, so they
 	# need to use more, smaller pages. if $cid is 0, then we get the
 	# totalviskids for the story 		--Pater
-	my $total = ($user->{mode} eq 'flat' || $user->{mode} eq 'nested')
-		? $comments->{$cidorpid}{totalvisiblekids}
-		: $cc;
-
-	my $lcp = $discussion2
```

Comment:
you took out too much here.  this is the correct line. At least for now until we figure out pagination.

my $lcp = linkCommentPages($discussion->{id}, $pid, $cid, $total);